stl/inc/atomic

```diff
@@ -354,8 +354,18 @@ struct _Atomic_storage {
         _Check_memory_order(_Order);
         const auto _Storage_ptr  = _STD addressof(_Storage);
         const auto _Expected_ptr = _STD addressof(_Expected);
+#if _HAS_CXX20
+        if constexpr (!_STD has_unique_object_representations_v<_Ty>) {

```

Comment:
Only functions need to be `_STD` qualified (because only functions activate Argument-Dependent Lookup). As a variable template, `has_unique_object_representations_v` can and should be mentioned unqualified.

(The only times we qualify non-functions are when we're in the "wrong" namespace, or if there's an ambiguity otherwise - which sometimes occurs with members.)

Occurs below.